server/apps/immich/src/api-v1/user/dto/create-user.dto.ts

```diff
@@ -1,27 +1,28 @@
+import { ApiProperty } from '@nestjs/swagger';
 import { IsNotEmpty, IsOptional } from 'class-validator';
 
 export class CreateUserDto {
   @IsNotEmpty()
+  @ApiProperty({ default: 'testuser@email.com' })
   email!: string;
 
   @IsNotEmpty()
+  @ApiProperty({ default: 'password' })
   password!: string;
 
   @IsNotEmpty()
+  @ApiProperty({ default: 'John' })
   firstName!: string;
 
   @IsNotEmpty()
+  @ApiProperty({ default: 'Doe' })
   lastName!: string;
 
   @IsOptional()
-  profileImagePath?: string;
-
-  @IsOptional()
+  @ApiProperty({ default: false })
   isAdmin?: boolean;
```

Comment:
If `isAdmin` has a default value when the DTO is hydrated, then the type should probably be `isAdmin: boolean = false` (currently means `boolean | undefined`) which can be inferred as well, so `isAdmin = false`. I think OpenAPI might "understand" it and probably wouldn't need the `ApiProperty` decorator. Same with `shouldChangePassword`

Reply:
This is correct 
![image](https://user-images.githubusercontent.com/27055614/178004204-bc505fd1-cf9a-4f6f-88e0-5c902e7ddf37.png)
Result after changing like suggestion

Reply:
🤔 I just had a quick look at the code. Looks like `isAdmin` and `shouldChangePassword` shouldn't be in `CreateUserDto` since they are not used (also there wouldn't be any use for them in a request DTO). They would be in the `UserResponseDto` and would be required.